AsyncDisplayKit/ASNetworkImageNode.mm

```diff
@@ -76,17 +80,17 @@ - (instancetype)initWithCache:(id<ASImageCacheProtocol>)cache downloader:(id<ASI
   
   ASDisplayNodeAssert([downloader respondsToSelector:@selector(downloadImageWithURL:callbackQueue:downloadProgress:completion:)] || [downloader respondsToSelector:@selector(downloadImageWithURL:callbackQueue:downloadProgressBlock:completion:)], @"downloader must respond to either downloadImageWithURL:callbackQueue:downloadProgress:completion: or downloadImageWithURL:callbackQueue:downloadProgressBlock:completion:.");
   
-  _downloaderSupportsNewProtocol = [downloader respondsToSelector:@selector(downloadImageWithURL:callbackQueue:downloadProgress:completion:)];
+  _downloaderFlags.downloaderSupportsNewProtocol = [downloader respondsToSelector:@selector(downloadImageWithURL:callbackQueue:downloadProgress:completion:)];
```

Comment:
@rcancro this is a good question. In this case, we are allocating only a single bit to the unsigned int, which is a bit unusual and I believe will be the same size across architectures. We do however use this pattern in several other places in the codebase, and so it has proven to work in practice, although I think it is somewhat questionable to do the assignment without an explicit cast…